templates/emails/github.signup.hbs

```diff
@@ -8,13 +8,24 @@ Subject: Your Open Collective is on its way...
 <p>Hi!</p>
 {{/if}}
 
-<p> Thanks for signing up your GitHub repository: {{group.name}}. We are reviewing new requests as fast as we can. </p>
+<p> Thanks for signing up on Open Collective! Your collective is now live: <a href="{{config.host.website}}/{group.slug}">{{group.name}}</a>. While you wait for approval to start receiving donations, here's a quick guide to make the most out of it: </p>
 
-<p>While you wait, check out other <a href="https://opencollective.com/opensource">open source collectives</a> and see how they are spending the money they have raised.</p>
+<ol type="1">
+<li><strong>Make your collective your own.</strong> Login and hover over elements to edit them. </li>
+
+<li><strong>Add backers/sponsors to Github/Website.</strong> We'll submit a PR to your README to add backers & sponsors. Instructions to add your backers to a website <a href="https://github.com/OpenCollective/OpenCollective/wiki/Website-banner">here</a>.
+
+<li><strong>Start tracking expenses.</strong> Submit expenses so others can see your costs. It helps to raise awareness (and later, money).
+
+<li><strong>Spread the word.</strong> If you have a twitter account, you can connect it to your collective and automatically thank your backers. <a href="https://github.com/OpenCollective/OpenCollective/wiki/Twitter">Instructions</a>
+
```

Comment:
Instead, I'd rather highlight 3 main things to do on top and then in another section but in the same email, a section "Other things you can do".
Don't forget that this is an email going out to open source projects. Those are technical people. They are also in the process of asking for money. It is justified in my opinion to ask them to do work for it.